### ISSUE TITLE
Update variable and constraint names after rolling

### ISSUE THREAD

**Issue by manuelma** [spine-tools/SpineOpt.jl#411]
I noticed that we set the variable and constraint names at the beginning of the first window, to a string that includes the TimeSlice index. When TimeSlices roll, obviously this string doesn't change automatically which means variables and constraints will have obsolete names. This may be misleading to people trying to debug models that rolled -> all the variables and constraints will have names with time slices from the first window.

**Comment by DillonJ** [comment]
Ah - is there a cheap way to fix this behaviour?

**Comment by DillonJ** [comment]
I was referring to performance rather than coding effort :-)

**Comment by clizbe** [comment]
Closing due to stale status.

**Comment by Tasqu** [comment]
I don't think we include time slice indices in any variable or constraint names anymore? TimeSlices are included in the indices, sure, but they roll with the window nowadays. I believe this is solved, or at least stale.
